app/pathogen/sarscov2/dashboard/(map)/SarsCov2Map.tsx

```diff
@@ -0,0 +1,99 @@
+"use client";
+
+import { Card, CardContent } from "@/components/ui/card";
+import React, { useContext, useState } from "react";
+import { PathogenMap } from "@/components/ui/pathogen-map/pathogen-map";
+import { useNewSarsCov2Data } from "@/hooks/useNewSarsCov2Data";
+import { SarsCov2Context } from "@/contexts/pathogen-context/pathogen-contexts/sc2-context";
+import { MapShadingLegend } from "@/app/pathogen/arbovirus/dashboard/(map)/MapShadingLegend";
+import { MapEstimateSummary } from "@/components/ui/pathogen-map/map-estimate-summary";
+import { MapSymbology } from "./map-config";
+
+export function SarsCov2Map() {
+  const state = useContext(SarsCov2Context);
+  const { data } = useNewSarsCov2Data();
+
+  if (!data) {
+    return <span> Loading... </span>;
+  }
+
+  return (
+    <>
+      <div className={"w-full h-full p-0"}>
+        <PathogenMap
+          id="sarsCov2Map"
+          baseCursor=""
+          sourceId="sc2-[GENERATED-SOURCE-ID]"
+          layers={[
+            {
+              id: "SARS-CoV2-pins",
+              type: "circle",
+              isDataUsedForCountryHighlighting: true,
```

Comment:
Country modal isn't a part of this PR.

https://github.com/serotracker/dashboard/issues/304